src/main/java/com/jfrog/ide/idea/ui/JFrogLocalToolWindow.java

```diff
@@ -225,7 +225,7 @@ private void setLeftPanelContent(JComponent component) {
     }
 
     private JBCefBrowser initVulnerabilityInfoBrowser(@NotNull Project project) throws IOException, URISyntaxException {
-        webviewManager = new WebviewManager(project);
+        webviewManager = new WebviewManager(project, () -> updateIssueOrLicenseInWebview(selectedIssue));
         Disposer.register(this, webviewManager);
```

Comment:
Let's clarify the reason behind passing the `selectedIssue` parameter during the initialization of the vulnerability info.